docs/user/gateways.md

```diff
@@ -0,0 +1,41 @@
+# Gateways
+
+Both the traces and the metrics feature are based on a gateway, which is provisioned as soon as you define any Pipeline resource. All telemetry data of the related domain passes the gateway, so it acts as a central point and provides the following benefits:
+- [Data Enrichment](#data-enrichment) to achieve a certain data quality
+- Filtering to apply namespace filtering and remove noisy system data (individually per domain)
+- Dispatching to the configured backends (individually per domain)
+
+When the Istio module is available, the gateway supports mTLS for the communication from the workload to the gateway, as well as for communication to backends running in the cluster; see [Istio support](#istio-support).
```

Comment:
```suggestion
When the Istio module is available, the gateways support mTLS for the communication from the workload to the gateway, as well as for communication to backends running in the cluster; see [Istio support](#istio-support).
```
Considering that we talk about both gateways (traces and metrics), I think plural is better here. Singular might imply that traces and metrics share a common gateway. With that, we're also consistent with the wording in line 21.